src/applications/disability-benefits/all-claims/constants.js

```diff
@@ -141,3 +141,21 @@ export const PTSD = 'ptsd';
 export const NINE_ELEVEN = '2001-09-11';
 
 export const ERR_MSG_CSS_CLASS = '.usa-input-error-message';
+
+export const submissionStatuses = {
+  // Statuses returned by the API
```

Comment:
Does this duplicate the constants in Increase Only? Should we just point those refs to the All Claims constants file and remove those old ones?